ddtrace/appsec/_processor.py

```diff
@@ -180,10 +181,11 @@ def delayed_init(self) -> None:
                 self._ddwaf = DDWaf(
                     self._rules, self.obfuscation_parameter_key_regexp, self.obfuscation_parameter_value_regexp
                 )
-                self.metrics._set_waf_init_metric(self._ddwaf.info, bool(self._ddwaf._handle))
+                self.metrics._set_waf_init_metric(self._ddwaf.info, self._ddwaf.initialized)
         except Exception:
             # Partial of DDAS-0005-00
-            log.warning("[DDAS-0005-00] WAF initialization failed")
+            log.error("[DDAS-0005-00] WAF initialization failed")
```

Comment:
why this change?

Reply:
yes, I should revert that. We can always do it in a another small PR if required.

Reply:
revert merged